packages/react-components/react-charts-preview/library/src/components/LineChart/LineChart.tsx

```diff
@@ -816,27 +827,10 @@ export const LineChart: React.FunctionComponent<ILineChartProps> = React.forward
           </g>,
         );
       }
-      const classes = useLineChartStyles_unstable(props);
-      // Removing un wanted tooltip div from DOM, when prop not provided.
-      if (!props.showXAxisLablesTooltip) {
```

Comment:
where is the code eq to this logic when showXAxisLabelsTooltip is false

Reply:
we do not need that as we are setting the props only when showXAxisLablesTooltip is true. Earlier it was removing the div from dom for which this block was kept.